content/concepts/tokens/non-fungible-token-transfers.md

```diff
@@ -0,0 +1,82 @@
+---
+html: non-fungible-token-transfers.html
+parent: non-fungible-tokens.html
+blurb: Trading NFTokens in direct or brokered mode.
+filters:
+ - include_code
+labels:
+ - Non-fungible Tokens, NFTs
+status: not_enabled
+---
+
+# Trading NFTokens on the XRP Ledger
+{% include '_snippets/nfts-disclaimer.md' %}
+
+You can transfer NFTokens between accounts on the XRP Ledger. You can transfer `NFTokens` by offering to buy or sell `NFTokens`, or accepting offers from other accounts. You can even give away `NFTokens` by offering to sell them at a price of 0.  All offers are created using the `NFTokenCreateOffer` transaction.
+
+
+## Sell Offers
+
+
+### Create a Sell Offer
+
+As the owner of a `NFToken object`, you can create a sell offer using a `NFTokenCreateOffer` transaction with a _Flags_ setting of _1_. You provide the _NFTokenID_ and the _Amount_ you are willing to accept in payment. You can optionally specify an _Expiration_ date, after which the offer is no longer valid, and a _Destination_ account, which is the only account that is allowed to purchase the `NFToken`.
+
+
+### Accept a Sell Offer
+
+To purchase a `NFToken` that is offered for sale, you use a `NFTokenAcceptOffer` transaction. You provide the owner account and specify the `nft_offer_index` of the `NFTokenOffer` object you choose to accept.
+
+
+## Buy Offers
+
+
+### Create a Buy Offer
+
+Any account can offer to buy a `NFToken` on the XRP Ledger. You can create a buy offer using `NFTokenCreateOffer` with a _Flags_ setting of _null_. You provide the _Owner_ account, _NFTokenID_, and the _Amount_ of your offer.
```

Comment:
It's not _quite_ accurate to say "a _Flags_ setting of _null_".

- `null` is a specific data type in JSON, which is not allowed for the Flags field in the `rippled` APIs. You can omit the field when constructing a transaction via JSON, which will cause it to be provided with a default value (possibly `0` though that depends on the interface).
- It is possible to specify other flags in the `Flags` field including global flags (currently the only one is tfFullyCanonicalSig, but this could change in the future), and still have the NFTokenCreateOffer treated as a buy.

I would say the most accurate way to phrase it would be either "with no NFTokenCreate Offer flags" or "without the `tfSellToken` flag".

Also, my notes on monospace font also apply here. And you can probably omit "on the XRP Ledger" since that's understood from context.

```suggestion
Any account can offer to buy a `NFToken`. You can create a buy offer using [NFTokenCreateOffer][] _without_ the `tfSellToken` flag. You provide the `Owner` account, `NFTokenID`, and the `Amount` of your offer.
```

Reply:
For whatever reason, in working with the call I found that omitting the flags field or setting it to 0 caused an error. Only setting the flag to null worked (and it continues to work in the code sample). We can come back to this at some future point.